ChustaSoft.Tools.DBAccess.MongoDb.IntegrationTests/RepositoryTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿using ChustaSoft.Tools.DBAccess.MongoDb.IntegrationTests.Base;
+using ChustaSoft.Tools.DBAccess.MongoDb.IntegrationTests.Models;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace ChustaSoft.Tools.DBAccess.MongoDb.IntegrationTests
+{
+    public class RepositoryTests : IntegrationTestBase
```

Comment:
Just for better identification, rename it to MongoRepositoryTests or something similar

Reply:
[Updated in linked commit](https://github.com/ChustaSoft/DBAccess/pull/22/commits/1379381d82883ab6f5fc38a749e797adb27b1917)